samples/nrf9160/modem_shell/src/link/link_shell_pdn.c

```diff
@@ -337,25 +337,30 @@ int link_shell_pdn_activate(int pdn_cid)
 	return ret;
 }
 
-void link_shell_pdn_init(void)
+void link_shell_pdn_init_callbacks(void)
```

Comment:
*nit* Why the function name uses a plural, when it only sets one callback? Would something like `link_shell_pdn_events_subscribe()` be more descriptive?

Reply:
Yes. Changed.